solarforecastarbiter/metrics/probabilistic.py

```diff
@@ -529,6 +544,51 @@ def continuous_ranked_probability_score(obs, fx, fx_prob):
     return crps
 
 
+def crps_skill_score(obs, fx, fx_prob, ref, ref_prob):
+    """CRPS skill score.
+
+        CRPSS = 1 - CRPS_fx / CRPS_ref
+
+    where CRPS_fx is the CPRS of the evaluated forecast and CRPS_ref is the
+    CRPS of a reference forecast.
+
+    Parameters
+    ----------
+    obs : (n,) array_like
+        Observations (physical unit).
+    fx : (n, d) array_like
+        Forecasts (physical units) of the right-hand-side of a CDF with d
+        intervals (d >= 2), e.g., fx = [10 MW, 20 MW, 30 MW] is interpreted as
+        <= 10 MW, <= 20 MW, <= 30 MW.
+    fx_prob : (n,) array_like
+        Probability [%] associated with the forecasts.
+    ref : (n, d) array_like
+        Reference forecasts (physical units) of the right-hand-side of a CDF
+        with d intervals (d >= 2), e.g., fx = [10 MW, 20 MW, 30 MW] is
+        interpreted as <= 10 MW, <= 20 MW, <= 30 MW.
+    ref_prob : (n,) array_like
+        Probability [%] associated with the reference forecast.
+
+    Returns
+    -------
+    skill : float
+        The CRPS skill score [unitless].
+
+    See Also
+    --------
+    :py:func:`solarforecastarbiter.metrics.probabilistic.continuous_ranked_probability_score`
+
+    """
+
+    if np.isscalar(ref):
+        return np.nan
+    else:
+        print("Valid ref:", ref, type(ref))
+        crps_fx = continuous_ranked_probability_score(obs, fx, fx_prob)
+        crps_ref = continuous_ranked_probability_score(obs, ref, ref_prob)
+        return 1 - crps_fx / crps_ref
```

Comment:
is it worth following the same "avoid divide by 0" pattern as in `deterministic.forecast_skill`? Seems less likely here, so I'm really just raising the question

Reply:
I do agree it's less likely, but I think it's worthwhile since it's a trivial change that adds robustness.